modules/nw-sriov-example-dpdk-line-rate.adoc

```diff
@@ -0,0 +1,278 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-example-dpdk-line-rate_{context}"]
+= Using SR-IOV and the Node Tuning Operator to achieve a DPDK line rate
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the SR-IOV Network Operator.
+* Log in as a user with `cluster-admin` privileges.
+* Deploy a standalone Node Tuning Operator.
+
+The Node Tuning Operator is used to configure isolated CPUs, hugepages, and a topology scheduler.
```

Comment:
Please move this module introduction and node to be prior to the `.Prerequisites` heading.

Reply:
Done.